File: Ejercicios Propuestos 3/Ejercicio 15/Ejercicio15.java
import java.util.Scanner;

public class Ejercicio15 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n, f, c;
        boolean es_simetrica;

        System.out.println("Ingrese el tamaño (n) de la matriz cuadrada:");
        n = sc.nextInt();

        int[][] matriz = new int[n][n];

        for (f = 0; f <= n - 1; f++) {
            for (c = 0; c <= n - 1; c++) {
                System.out.println("Valor para la posición [" + f + "][" + c + "]:");
                matriz[f][c] = sc.nextInt();
            }
        }

        es_simetrica = true;

        for (f = 0; f <= n - 1; f++) {
            for (c = 0; c <= n - 1; c++) {
                if (matriz[f][c] != matriz[c][f]) {
                    es_simetrica = false;
                }
            }
        }

        System.out.println("MATRIZ:");
        for (f = 0; f <= n - 1; f++) {
            for (c = 0; c <= n - 1; c++) {
                System.out.print(matriz[f][c] + "  ");
            }
            System.out.println("");
        }

        System.out.println("--------------------------------------------------");
        if (es_simetrica) {
            System.out.println("La matriz es SIMÉTRICA.");
        } else {
            System.out.println("La matriz NO es simétrica.");
        }
        sc.close();
    }
}

File: Ejercicios Propuestos 3/Ejercicio 15/Ejercicio 15.py
NAME = "Renzo Leonardo Cienfuegos Cardenas"
GROUP = 2

n = int(input("Ingrese el tamaño (n) de la matriz cuadrada: "))

matriz = []

for f in range(n):
    fila = []
    for c in range(n):
        valor = int(input(f"Valor para la posición [{f}][{c}]: "))
        fila.append(valor)
    matriz.append(fila)

es_simetrica = True

for f in range(n):
    for c in range(n):
        if matriz[f][c] != matriz[c][f]:
            es_simetrica = False

print("MATRIZ:")
for f in range(n):
    for c in range(n):
        print(matriz[f][c], end="  ")
    print()

print("--------------------------------------------------")

if es_simetrica:
    print("La matriz es SIMÉTRICA.")
else:
    print("La matriz NO es simétrica.")
